app/views/support/BillingScheduleOps.scala

```diff
@@ -5,6 +5,6 @@ import scalaz.syntax.std.boolean._
 object BillingScheduleOps {
   implicit class InvoiceOps(bill: Bill) {
     def isDiscounted(str: String): Option[String] =
-      bill.items.list.exists(_.amount < 0).option(str)
+      (bill.accountCredit.isDefined || bill.items.list.exists(_.amount < 0)).option(str)
```

Comment:
I'd prefer to separate the boolean determining logic from the massaging types, so even if we want to keep the behaviour of this method, put it into a val and then use the val on the next line